src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -400,6 +421,41 @@ public override bool Execute()
         }
     }
 
+    private bool ProcessNettrace(string netTraceFile)
+    {
+        var outputMibcPath = Path.Combine(OutputDir, Path.ChangeExtension(Path.GetFileName(netTraceFile), ".mibc"));
+
+        if (_cache!.Enabled && File.Exists(outputMibcPath))
+        {
+            string hash = Utils.ComputeHash(netTraceFile);
+            if (!_cache!.UpdateAndCheckHasFileChanged($"-mibc-source-file-{Path.GetFileName(netTraceFile)}", hash))
+            {
+                Log.LogMessage(MessageImportance.Low, $"Skipping generating {outputMibcPath} from {netTraceFile} because source file hasn't changed");
+                return true;
+            }
+            else
+            {
+                Log.LogMessage(MessageImportance.Low, $"Generating {outputMibcPath} from {netTraceFile} because the source file's hash has changed.");
+            }
+        }
+
+        (int exitCode, string output) = Utils.TryRunProcess(Log,
+                                                            PgoBinaryPath!,
+                                                            $"create-mibc --trace {netTraceFile} --output {outputMibcPath}",
+                                                            silent: false,
+                                                            debugMessageImportance: MessageImportance.Low);
```

Comment:
These are the default values for the arguments, so you don't need to pass them.

Reply:
Done!